rinstall.sh

```diff
@@ -36,46 +36,51 @@ case $(dirname "$target") in
 		;;
 esac
 
-if test ! -f "$1"; then
+if [ ! -f "$1" ]; then
 	umask 044
 	source="$(basename "$1" | tr -C '0-9a-zA-Z_' '_')$(mktemp XXXXXX)"
-	case `uname` in
+        case $(uname) in
 		OpenBSD|NetBSD)
 			ftp -o "$source" -n "$INSTALL_URL/$1"
 			;;
 		FreeBSD|DragonFly)
-			fetch -q -o "$source" "$INSTALL_URL/$1"
+			fetch -qo "$source" "$INSTALL_URL/$1"
 			;;
 		Linux|Darwin|SunOS)
-			if command -pv wget > /dev/null; then
-				wget -q -O "$source" "$INSTALL_URL/$1"
+			if command -pv curl > /dev/null; then
+				curl -fsSLo "$source" "$INSTALL_URL/$1"
 			else
-				curl -f -s -o "$source" "$INSTALL_URL/$1"
+				wget -qO "$source" "$INSTALL_URL/$1"
 			fi
 			;;
 		*)
-			echo "Unknown OS"; exit 2
+			>&2 echo "Unknown OS"
+                        exit 2
```

Comment:
Mix of tabs and spaces, but this is minor and can be fixed in the next commit

Reply:
Sure, I'll fix that next time.
Thanks for catching this. I thought, I replaced back all spaces that were accidentally added ;)